infra/gcp/lib_util.sh

```diff
@@ -35,7 +38,7 @@ function _nocolor() {
 function color() {
     _color "$1"
     shift
-    echo "$@$(_nocolor)"
+    echo "$*$(_nocolor)"
```

Comment:
I am curious why this?

Reply:
I don't feel strongly about this, but my editor is setup to nag me about shellcheck errors (vs. warnings), and this came up as: https://github.com/koalaman/shellcheck/wiki/SC2145

Reply:
nteresting read

Reply:
FWIW, I'm now seeing colorized output on my mac (running bash 4.4 installed by homebrew).  I suspect this was the change that did it.

Reply:
Nice.